WooCommerce/WooCommerceTests/Mocks/MockProductReviewFromNoteParcel.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import XCTest
+@testable import Yosemite
```

Comment:
`@testable` is not needed and probably misleading. 🙂 

```suggestion
import XCTest
import Yosemite
```

Reply:
Updated in: https://github.com/woocommerce/woocommerce-ios/pull/5044/commits/15d1131bbe5e1c8d293066161c905502a76d48e5